src/components/button/button.ts

```diff
@@ -104,8 +106,8 @@ export class MButton extends Vue {
         return this.iconPosition == MButtonIconPosition.Right && this.waiting;
     }
 
-    private get hasPrecisionSlot(): boolean {
-        return !!this.$slots.precision;
+    private get hasPrecision(): boolean {
+        return (this.precision != '' && this.precision != undefined) || !!this.$slots.precision;
```

Comment:
cette ligne pourrait être remplacée par `(!!this.precision || !!this.$slots.precision;`